docs/ossm/kiali/README.md

```diff
@@ -0,0 +1,463 @@
+
+* **Cluster API** - Kiali uses the API of the OpenShift (cluster API) to fetch and resolve service mesh configurations. Kiali queries the cluster API to retrieve, for example, definitions for namespaces, services, deployments, pods, and other entities. Kiali also makes queries to resolve relationships between the different cluster entities. The cluster API is also queried to retrieve Istio configurations like virtual services, destination rules, route rules, gateways, quotas, and so on.
+
+* **Tempo (optional)** - When you install Tempo, the Kiali console includes a tab to display distributed tracing data. Note that tracing data will not be available if you disable Istio's distributed tracing feature. Also note that user must have access to the namespace where the Istio control plane is installed to view tracing data.
+
+* **Grafana (optional)** - When available, the metrics pages of Kiali display links to direct the user to the same metric in Grafana. Note that user must have access to the namespace where the Istio control plane is installed to view links to the Grafana dashboard and view Grafana data.
+
+### Kiali Features
+
+The Kiali console is integrated with Red Hat Service Mesh and provides the following capabilities:
+
+* **Health** – Quickly identify issues with applications, services, or workloads.
+
+* **Traffic** – Visualize how your applications, services, or workloads communicate via the Kiali traffic graph.
+
+* **Metrics** – Predefined metrics dashboards let you chart service mesh and application performance for Go, Node.js. Quarkus, Spring Boot, Thorntail and Vert.x. You can also create your own custom dashboards.
+
+* **Tracing** – Integration with Jaeger lets you follow the path of a request through various microservices that make up an application.
+
+* **Validations** – Perform advanced validations on the most common Istio objects (Destination Rules, Service Entries, Virtual Services, and so on).
+
+* **Mesh structure** – Detailed information about the Istio infrastructure status is displayed on the mesh page. It shows an infrastructure topology view with core and add-on components, their health, and how they are connected to each other.
+
+* **Configuration** – Optional ability to create, update and delete Istio routing configuration using wizards or directly in the YAML editor in the Kiali Console.
+
+## Installing the Kiali
+
+Kiali can be installed in two different ways: via the OpenShift web console or the OpenShift CLI.
+
+### Via the OpenShift web console
+
+The following steps show how to install the Kiali via the OpenShift web console.
+
+[!WARNING]
+Do not install the Community version of the Operator. The Community version is not supported.
+
+**Prerequisites**
+
+* Access to the OpenShift web console.
```

Comment:
Need not just access, but admin access because installing operators requires that.

```suggestion
* Access to the OpenShift web console with administrator access
```